presto-tests/src/test/java/com/facebook/presto/tests/TestSqlFunctions.java

```diff
@@ -50,6 +50,7 @@ private static QueryRunner createQueryRunner()
                     .setSchema(TINY_SCHEMA_NAME)
                     .build();
             DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(session)
+                    .setExtraProperties(ImmutableMap.of("inline-sql-functions", "false"))
```

Comment:
Why are we disabling inline sql functions here?

Reply:
Cause the bug only exists if we compile SQL function at execution time. This class is in general written to test SQL function. Inlined SQL functions are pretty much just macros and they work as normal expressions during most of planning and runtime. Thus I just disabled this by default. The inline sql function tests are in `TestInlineSqlFunctions`.